src/utils/report.js

```diff
@@ -57,6 +57,7 @@ export default function ({
 
   const severity = get(result.stylelint, [ "ruleSeverities", ruleName ], "ignore")
 
+  /* istanbul ignore if */
   if (typeof severity === "undefined") {
```

Comment:
Why should this be ignored in code coverage? We have the error there because it happened before and we wanted to ensure we had a friendly message if it happened again.